interfaces/Glitter/templates/include_queue.tmpl

```diff
@@ -67,8 +67,10 @@
             <tr>
                 <th style="width: 25px;"></th>
                 <th></th>
-                <th class="table-header-extra" data-bind="css: { 'table-extra-header-visible' : extraQueueColumn }"></th>
-                <th class="table-header-progress" data-bind="css: { 'table-header-progress-smaller' : extraQueueColumn }"></th>
+                <!-- ko foreach: extraQueueColumns -->
+                <th style="width:7%"></th>
```

Comment:
I would suggest maybe `width: 5%`, 7 seems a lot.

Reply:
5% seemed very narrow in my tests so I set it to 6. Other than that I think I've done the changes you suggested.